app/docker/helpers/containerHelper.js

```diff
@@ -213,6 +220,7 @@ angular.module('portainer.docker').factory('ContainerHelper', [
             const portKeySplit = _.split(portKey, '/');
             const containerPort = parseInt(portKeySplit[0]);
             const portBinding = portBindings[portKey][0];
+            portBindings[portKey].shift();
```

Comment:
why is that needed?

Reply:
Without removing the first element from the array (which can have many elements), we are always returning the first element resulting in the errors I have shown above